rg_instructor_analytics_log_collector/tests/processors/test_course_activity_pipeline.py

```diff
@@ -0,0 +1,49 @@
+"""
+Test `CourseActivityPipeline` functionality.
+"""
+import logging
+from unittest import skip, TestCase
+
+from ddt import ddt, data, file_data, unpack
+from mock import patch
+
+from rg_instructor_analytics_log_collector.processors.course_activity_pipeline import CourseActivityPipeline
+from rg_instructor_analytics_log_collector.processors.course_activity_pipeline import CourseKey
+from rg_instructor_analytics_log_collector.tests.processors.pipeline_test_utils import TestRecord
+
+
+@ddt
+class TestCourseActivityPipeline(TestCase):
+    """
+    Test `CourseActivityPipeline` logic.
+    """
+
+    def setUp(self):
+        logging.disable(logging.DEBUG)
+        self.pipeline = CourseActivityPipeline()
+
+    @file_data("test_resources/test_course_activity_pipeline_records.json")
+    @unpack
+    def test_is_valid(self, entry):
+        self.assertEqual(self.pipeline.is_valid(entry.get("data")), entry.get("is_valid"))
+
+    @skip("Skip till figure out a reason")
```

Comment:
These are the first tests in the app and even they are skipped =) I have a feeling that this will stay here forever after it is merged, so let's either remove the skipped tests or fix them.

Reply:
The issue couldn't be reproduced anymore; tests run fine